app/models/proposal.rb

```diff
@@ -160,12 +169,30 @@ def on_rejected_entry(prev_state, event)
 
   def restart
     # Note that none of the state machine's history is stored
+    # Further note: this isn't used anywhere
```

Comment:
My bad. I'll remove the comment and add a ticket to evaluate - I'm guessing I'm not the only one who thought NCR and 18f operated the same way when editing.